### PR TITLE
engine: added audio backend print to s_info command

### PR DESCRIPTION
**Pull request by SNMetamorph**

### CODE REVIEW

File: engine/client/s_main.c [hunk 1]
```diff
@@ -1927,6 +1927,7 @@ qboolean S_Init( void )
 	Cmd_AddCommand( "spk", S_SayReliable_f, "reliable play a specified sententce" );
 	Cmd_AddCommand( "speak", S_Say_f, "playing a specified sententce" );
 
+	dma.backendName = NULL;
```
Comment (a1batross) [inline engine/client/s_main.c:1930]: Do you really need to set it to NULL beforehand?

File: engine/platform/sdl/s_sdl.c [hunk 2]
```diff
@@ -141,6 +141,7 @@ qboolean SNDDMA_Init( void )
 	Con_Printf( "Using SDL audio driver: %s @ %d Hz\n", SDL_GetCurrentAudioDriver( ), obtained.freq );
 
 	dma.initialized = true;
+	dma.backendName = "SDL";
```
Comment (a1batross) [inline engine/platform/sdl/s_sdl.c:144]: For SDL, we probably can write something like "SDL (%s)", where %s is SDL_GetCurrentAudioDriver output.

File: engine/client/s_main.c (hunk 1 above)
Comment (SNMetamorph) [reply engine/client/s_main.c:1930]: I think better to do it, that's will protect if dma.backendName won't set in some of audio backends, for example

File: engine/platform/sdl/s_sdl.c (hunk 2 above)
Comment (SNMetamorph) [reply engine/platform/sdl/s_sdl.c:144]: Yes, but then we need create char array for that in s_sdl.c

Comment (SNMetamorph) [reply engine/platform/sdl/s_sdl.c]: I did it

File: engine/client/s_main.c (hunk 1 above)
Comment (a1batross) [reply engine/client/s_main.c:1930]: @SNMetamorph we better not have such backends at all. :)

Also, globals are zero initialized anyway by standard.

Comment (SNMetamorph) [reply engine/client/s_main.c:1930]: Ok

Comment (nekonomicon) [reply engine/client/s_main.c:1930]: May be set "None" as backend by default?

Comment (SNMetamorph) [reply engine/client/s_main.c:1930]: Yes

**Review by a1batross — APPROVED** [review]

**Pull request merged**